backend/molgenis-emx2-datamodels/src/main/java/org/molgenis/emx2/datamodels/PetStoreLoader.java

```diff
@@ -54,14 +51,16 @@ public void load(Schema schema, boolean includeDemoData) {
         table(ORDER)
             .add(column(ORDER_ID).setPkey())
             .add(column("pet").setType(REF).setRefTable(PET))
-            .add(column(QUANTITY).setType(LONG).setValidation("{quantity} >= 1"))
-            .add(column(PRICE).setType(DECIMAL).setValidation("{price} >= 1"))
+            .add(
+                column(QUANTITY)
+                    .setType(LONG)
+                    .setValidation("if(quantity < 1) 'quantity should be >= 1'"))
```

Comment:
this is an invalid validation for type long (since it is actually a string)

Reply:
> Looks cool, we may want to gray out the computed field in data editor (like name). It now gives the suggestion you can change the computed value.

I thought we had a readonly on line 17. If that doesn't work then something is wrong :-)

Reply:
it strangely enough seems to work?